src/Exception/TopicDoesNotExist.php

```diff
@@ -0,0 +1,14 @@
+<?php
+declare(strict_types=1);
+
+namespace Kafka\Exception;
+
+use Kafka\Exception;
+
+final class TopicDoesNotExist extends Exception
+{
+    public function __construct(string $topic)
```

Comment:
Please don't override the constructor, we lose some important features related to exceptions (`$previous`). You can use a named constructor to achieve the same result without compromising that.